docs/blueprints/EP001.rst

```diff
@@ -1,3 +1,14 @@
+:EP: BP001
```

Comment:
Using both EP and BP for the ID is confusing. Also, some files have three digits, others two. I suggest leaving just the number, without padding:
```suggestion
:EP: 1
```

Reply:
Ok, I updated the others BP too.